docs/components/board/_index.md

```diff
@@ -99,1209 +99,9 @@ import (
-
-{{% /tab %}}
-{{< /tabs >}}
-
-## `Analog` API
-
-### Read
-
-Read the current value from an analog pin or analog reader capable of reading analog values.
-
-{{< tabs >}}
-{{% tab name="Python" %}}
-
-**Parameters:**
-
-- `extra` [(Optional\[Dict\[str, Any\]\])](https://docs.python.org/library/typing.html#typing.Optional): Extra options to pass to the underlying RPC call.
-- `timeout` [(Optional\[float\])](https://docs.python.org/library/typing.html#typing.Optional): An option to set how long to wait (in seconds) before calling a time-out and closing the underlying RPC call.
-
-**Returns:**
-
-- [(Analog.Value)](https://python.viam.dev/autoapi/viam/components/board/index.html#viam.components.board.Board.Analog.Value): The result of reading an analog reader.
-  It contains the raw data read as `value`, the reader’s minimum and maximum possible values as `min_range` and `max_range`, and its `step_size` (the minimum possible change between values it can read).
-
-For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/board/index.html#viam.components.board.Board.Analog.read).
-
-```python
-my_board = Board.from_robot(robot=robot, name="my_board")
-
-# Get the GPIOPin with pin number 15.
-pin = await my_board.gpio_pin_by_name(name="15")
-
-# Get if it is true or false that the pin is set to high.
-duty_cycle = await pin.get_pwm()
-
-# Get the Analog pin "my_example_analog".
-analog = await my_board.analog_by_name(
-    name="my_example_analog")
-
-# Get the value of the digital signal "my_example_analog" has most
-# recently measured.
-reading = analog.read()
-reading_value = reading.value
-```
-
-{{% /tab %}}
-{{% tab name="Go" %}}
-
-**Parameters:**
-
-- `ctx` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
-- `extra` [(map\[string\]interface{})](https://go.dev/blog/maps): Extra options to pass to the underlying RPC call.
-
-**Returns:**
-
-- [(AnalogValue)](https://pkg.go.dev/go.viam.com/rdk/components/board#AnalogValue): The current value, including the integer `Value` of the digital signal output by the analog pin and the `Min`, `Max`, and `StepSize` of the reader.
-- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
-
-For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/components/board#Analog).
-
-```go
-myBoard, err := board.FromRobot(robot, "my_board")
-
-// Get the analog pin "my_example_analog".
-analog, err := myBoard.AnalogByName("my_example_analog")
-
-// Get the value of the digital signal "my_example_analog" has most recently measured.
-reading, err := analog.Read(context.Background(), nil)
-readingValue := reading.Value
-stepSize := reading.StepSize
-```
-
-{{% /tab %}}
-{{< /tabs >}}
-
-### Write
-
-Write an analog value to a pin or analog output on a board capable of doing so.
-
-{{< tabs >}}
-{{% tab name="Python" %}}
```

Comment:
This went missing I think? Unless no longer in use

Reply:
This one is malformed upstream, in a manner that usually reflects purposeful deprecation or an unimplemented method. I'll try to track down what/why after this PR.